CakePHP/Tests/Commenting/InheritDocUnitTest.1.inc

```diff
@@ -29,6 +29,15 @@ class Foo extends Bar
      */
     public $hand = [];
 
+    /**
+     * Short description first.
+     *
+     * {@inheritDoc}
```

Comment:
The unfixed and fixed docblocks look the same to me.

Reply:
I couldn't auto-fix everything because don't know what's supposed to be. Maybe I'll split it into multiple files later if I add more.